impl/sql/mutationstorage/queue_test.go

```diff
@@ -16,53 +16,81 @@ package mutationstorage
 
 import (
 	"context"
+	"sync"
 	"testing"
 	"time"
 
 	"github.com/google/keytransparency/core/mutator"
-)
 
-var (
-	min = time.Millisecond * 2
-	max = time.Millisecond * 10
+	pb "github.com/google/keytransparency/core/api/v1/keytransparency_proto"
 )
 
-func TestRecieverTime(t *testing.T) {
+func TestRecieverChan(t *testing.T) {
 	ctx := context.Background()
 	m, err := New(newDB(t))
 	if err != nil {
 		t.Fatalf("Failed to create mutations: %v", err)
 	}
 
+	min := 10 * time.Hour
+	max := 60 * time.Hour
+	minC := make(chan time.Time)
+	maxC := make(chan time.Time)
+	fake := make(chan time.Time, 10)
 	now := time.Now()
-	for i, tc := range []struct {
-		want int
-		last time.Time
-		wait time.Duration
+	for _, tc := range []struct {
+		desc  string
+		send  int
+		start int64
+		last  time.Time
+		C     chan time.Time
+		want  int
```

Comment:
'want' is a bit general - wantNumCalls? wantNumMsgs?

Reply:
`wantCalls`